resources/knative-serving/templates/tests/test-serving.yaml

```diff
@@ -1,22 +1,83 @@
-#---
-#apiVersion: v1
-#kind: Pod
-#metadata:
-#  name: test-knative-serving-acceptance
-#  namespace: knative-serving
-#  annotations:
-#    "helm.sh/hook": test-success
-#  labels:
-#    "helm-chart-test": "true"
-#spec:
-#  containers:
-#  - name: test
-#    image: {{ .Values.global.containerRegistry.path }}/{{ .Values.global.test_knative_serving_acceptance.dir }}knative-serving-acceptance-tests:{{ .Values.global.test_knative_serving_acceptance.version }}
-#    env:
-#    - name: INGRESSGATEWAY_ADDRESS
-#      value: knative-ingressgateway.istio-system.svc.cluster.local
-#    - name: DOMAIN_NAME
-#      value: {{ .Values.global.ingress.domainName }}
-#    - name: TARGET
-#      value: {{ .Values.test.target }}
-#  restartPolicy: Never
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: knative-serving-test
+  labels:
+    helm-chart-test: "true"
+rules:
+- apiGroups: ["serving.knative.dev"]
+  resources: ["services"]
+  verbs: ["*"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: knative-serving-test
+  namespace: knative-serving
+  labels:
+    helm-chart-test: "true"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: knative-serving-test
+  namespace: knative-serving
+  labels:
+    helm-chart-test: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: knative-serving-test
+subjects:
+- kind: ServiceAccount
+  name: knative-serving-test
+  namespace: knative-serving
+---
+{{- if .Capabilities.APIVersions.Has "testing.kyma-project.io/v1alpha1" }}
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
+metadata:
+  name: test-knative-serving-acceptance
+template:
+  metadata:
+    annotations:
+      sidecar.istio.io/inject: "true"
+  spec:
```

Comment:
Containers shoulf be defined under: spec.template.spec, here you have an example:
```
apiVersion: testing.kyma-project.io/v1alpha1
kind: TestDefinition
metadata:
  name: test-sth
spec:
  template:
    spec:
      shareProcessNamespace: true
      restartPolicy: Never
      containers:
        - name: test
          image: alpine:latest
```

Reply:
ah, yes that make more sense